include/zephyr/xen/public/xen.h

```diff
@@ -26,8 +26,8 @@
  * Copyright (c) 2004, K A Fraser
  */
 
-#ifndef __XEN_PUBLIC_XEN_H__
-#define __XEN_PUBLIC_XEN_H__
+#ifndef ZEPHYR_INCLUDE_XEN_PUBLIC_XEN_H_
+#define ZEPHYR_INCLUDE_XEN_PUBLIC_XEN_H_
```

Comment:
[These are Xen headers were copied from Xen repository](https://github.com/zephyrproject-rtos/zephyr/commit/bfe9989d9f95a8b3ae2308698d24c2d76767d390).

We do not want to create unnecessary diffs like these for external headers.

Reply:
Oh, I see. I will roll back all changes to xen.

Reply:
But I have a question, that is, can I change the header files in the non-public directory under the xen directory?